query-engine/driver-adapters/src/proxy.rs

```diff
@@ -87,6 +100,20 @@ impl DriverProxy {
         .await
     }
 
+    pub async fn transaction_context(&self) -> quaint::Result<Box<JsTransactionContext>> {
```

Comment:
Why do we box it here? 56 bytes doesn't seem to be too large. Is it just for consistency with `start_transaction`?